styles/_mixin.scss

```diff
@@ -0,0 +1,98 @@
+
+
+@mixin Primary-btn {
+    border-radius:2px;
+    background: #397DF6;
+    color: #FFF;
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 24px;
+    cursor: pointer;
+  }
+@mixin Secondary-btn {
+    cursor: pointer;
+  }
+@mixin Outline-btn {
+  cursor: pointer;
+}
+@mixin Large {
```

Comment:
what is 'large'? how to use it? where to use it? not clear.
https://sass-lang.com/documentation/at-rules/mixin/